### ISSUE TITLE
0.1.3 - install generator fails uninitialized constant Administrate::Generators::InstallGenerator::DashboardManifest

### ISSUE THREAD

**Issue by yourabi** [thoughtbot/administrate#448]
rails generate administrate:install

```

I, [2016-01-30T17:45:36.035990 #21193]  INFO -- : ** [Raven] Raven 0.12.2 ready to catch errors
      create  app/controllers/admin/application_controller.rb
       route  namespace :admin do
    DashboardManifest::DASHBOARDS.each do |dashboard_resource|
      resources dashboard_resource
    end

    root controller: DashboardManifest::ROOT_DASHBOARD, action: :index
  end
      create  app/dashboards/dashboard_manifest.rb
/home/vagrant/.gem/ruby/2.3.0/gems/administrate-0.1.3/lib/generators/administrate/install/install_generator.rb:44:in `manifest': uninitialized constant Administrate::Generators::InstallGenerator::DashboardManifest (NameError)
	from /home/vagrant/.gem/ruby/2.3.0/gems/administrate-0.1.3/lib/generators/administrate/install/install_generator.rb:36:in `dashboard_resources'
	from /home/vagrant/.gem/ruby/2.3.0/gems/administrate-0.1.3/lib/generators/administrate/install/install_generator.rb:32:in `singular_dashboard_resources'
	from /home/vagrant/.gem/ruby/2.3.0/gems/administrate-0.1.3/lib/generators/administrate/install/install_generator.rb:24:in `run_dashboard_generators'
	from /home/vagrant/.gem/ruby/2.3.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /home/vagrant/.gem/ruby/2.3.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /home/vagrant/.gem/ruby/2.3.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `block in invoke_all'
	from /home/vagrant/.gem/ruby/2.3.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `each'
	from /home/vagrant/.gem/ruby/2.3.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `map'
	from /home/vagrant/.gem/ruby/2.3.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `invoke_all'
	from /home/vagrant/.gem/ruby/2.3.0/gems/thor-0.19.1/lib/thor/group.rb:232:in `dispatch'
	from /home/vagrant/.gem/ruby/2.3.0/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
	from /home/vagrant/.gem/ruby/2.3.0/gems/railties-4.2.5.1/lib/rails/generators.rb:157:in `invoke'
	from /home/vagrant/.gem/ruby/2.3.0/gems/railties-4.2.5.1/lib/rails/commands/generate.rb:13:in `<top (required)>'
	from /home/vagrant/.gem/ruby/2.3.0/gems/railties-4.2.5.1/lib/rails/commands/commands_tasks.rb:123:in `require_command!'
	from /home/vagrant/.gem/ruby/2.3.0/gems/railties-4.2.5.1/lib/rails/commands/commands_tasks.rb:130:in `generate_or_destroy'
	from /home/vagrant/.gem/ruby/2.3.0/gems/railties-4.2.5.1/lib/rails/commands/commands_tasks.rb:50:in `generate'
	from /home/vagrant/.gem/ruby/2.3.0/gems/railties-4.2.5.1/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
	from /home/vagrant/.gem/ruby/2.3.0/gems/railties-4.2.5.1/lib/rails/commands.rb:17:in `<top (required)>'
	from bin/rails:4:in `require'
	from bin/rails:4:in `<main>'

```

**Comment by yourabi** [comment]
I should add downgrading to 0.1.2 seems to work

**Comment by steffenix** [comment]
https://github.com/thoughtbot/administrate/pull/435

**Comment by coreyward** [comment]
Having the same issue. Tried to debug, but couldn't even get the Administrate specs to pass. Going back to ActiveAdmin, sadly.

**Comment by graysonwright** [comment]
I created #464 to discuss potential solutions to this. Please join in the conversation there! This is an important fix, so I'll package another release as soon as we fix it up.

**Comment by graysonwright** [comment]
Fixed by #464, which will go out in `0.2.0`.
